src/api/partner/content-types/partner/lifecycles.ts

```diff
@@ -0,0 +1,46 @@
+import type { Event } from '@strapi/database/dist/lifecycles'
+import { errors } from '@strapi/utils'
+
+const resourceType = 'api::partner.partner'
+
+const MAX_NUMBER_PARTNERS_DISPLAYED = 6
+
+const validateBanner = async (event: Event) => {
+  const {
+    data: { displayOnLandingPage, documentId },
+  } = event.params
+
```

Comment:
You can early return if not `displayOnLandingPage` here

Reply:
Oui bien vu !

Reply:
Je suis dessus. Il y avait des soucis avec la requête aussi